pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -238,10 +238,18 @@ type CARotation struct {
 	// LastInitiationTime is the most recent time when the certificate authority credential rotation was initiated.
 	// +optional
 	LastInitiationTime *metav1.Time `json:"lastInitiationTime,omitempty" protobuf:"bytes,3,opt,name=lastInitiationTime"`
+	// LastInitiationFinishedTime is the recent time when the certificate authority credential rotation initiated was
+	// completed.
+	// +optional
+	LastInitiationFinishedTime *metav1.Time `json:"lastInitiationFinishedTime,omitempty" protobuf:"bytes,4,opt,name=lastInitiationFinishedTime"`
```

Comment:
Should we not order fields here based on `protobuf` number? Otherwise, it can be confusing if in the future someone wants to add a new field, one need to go through protobuf no. of all fields instead of checking just the last one.
Also in general we sort fields based on protobuf number.